src/pages/api/trpc/[trpc].ts

```diff
@@ -1,9 +1,35 @@
 import * as trpcNext from '@trpc/server/adapters/next';
+import * as Sentry from '@sentry/nextjs';
 
 import { trpcRouter } from '../../../../trpc/router';
 import { createContext } from '../../../../trpc/context';
 
+const errorSubstitution = <T extends { code: string; message: string; stack?: string }>(
+    error: T,
+    code: 'BAD_REQUEST',
+) => {
+    error.code = code;
+    error.message = 'Write to us about the error';
```

Comment:
There should be either a support link or a more general message like 'Something went wrong'